src/sb-util.ts

```diff
@@ -179,7 +184,19 @@ export class Sprite extends SpriteCollection {
     }
 
     public blocks(query?: string): BlockCollection {
-        const blocks = new BlockCollection(makeIterable(this.prop('blocks'), deserializeBlocks));
+        const deserialized = makeIterable(this.prop('blocks'), deserializeBlocks);
+        const tagOwnerIterator = makeIterable(
+            deserialized,
+            (d): Iterator<BlockProperties> =>
+                map(
+                    d,
+                    (b): BlockProperties => {
+                        getBlockMeta(b).sprite = this.props();
```

Comment:
This is where the parent info gets stored

Reply:
Hmm. Future note: How do we apply this to a query on `project.query('block')` where we query all blocks.

Reply:
`project.blocks()` is a flatMap of `this.sprites(), s=>s.blocks()` so it still gets set.